api/envoy/config/filter/network/tcp_proxy/v2/tcp_proxy.proto

```diff
@@ -62,72 +52,6 @@ message TcpProxy {
-    // match.
-    message TCPRoute {
-      // The cluster to connect to when a the downstream network connection
-      // matches the specified criteria.
-      string cluster = 1 [(validate.rules).string.min_bytes = 1];
-
-      // An optional list of IP address subnets in the form
-      // “ip_address/xx”. The criteria is satisfied if the destination IP
-      // address of the downstream connection is contained in at least one of
-      // the specified subnets. If the parameter is not specified or the list
-      // is empty, the destination IP address is ignored. The destination IP
-      // address of the downstream connection might be different from the
-      // addresses on which the proxy is listening if the connection has been
-      // redirected.
-      repeated envoy.api.v2.core.CidrRange destination_ip_list = 2;
-
-      // An optional string containing a comma-separated list of port numbers
-      // or ranges. The criteria is satisfied if the destination port of the
-      // downstream connection is contained in at least one of the specified
-      // ranges. If the parameter is not specified, the destination port is
-      // ignored. The destination port address of the downstream connection
-      // might be different from the port on which the proxy is listening if
-      // the connection has been redirected.
-      string destination_ports = 3;
-
-      // An optional list of IP address subnets in the form
-      // “ip_address/xx”. The criteria is satisfied if the source IP address
-      // of the downstream connection is contained in at least one of the
-      // specified subnets. If the parameter is not specified or the list is
-      // empty, the source IP address is ignored.
-      repeated envoy.api.v2.core.CidrRange source_ip_list = 4;
-
-      // An optional string containing a comma-separated list of port numbers
-      // or ranges. The criteria is satisfied if the source port of the
-      // downstream connection is contained in at least one of the specified
-      // ranges. If the parameter is not specified, the source port is
-      // ignored.
-      string source_ports = 5;
-    }
-
-    // The route table for the filter. All filter instances must have a route
-    // table, even if it is empty.
-    repeated TCPRoute routes = 1 [(validate.rules).repeated .min_items = 1];
-  }
-
-  // TCP Proxy filter configuration using deprecated V1 format. This is required for complex
-  // routing until filter chain matching in the listener is implemented.
-  //
-  // .. attention::
-  //
-  //   Using this field will lead to `problems loading the configuration
-  //   <https://github.com/envoyproxy/envoy/issues/2441>`_. If you want to configure the filter
-  //   using v1 config structure, please make this field a boolean with value ``true`` and configure
-  //   via the opaque ``value`` field like is suggested in :api:`envoy/config/filter/README.md`.
-  DeprecatedV1 deprecated_v1 = 6 [deprecated = true];
```

Comment:
Please reserve this: https://developers.google.com/protocol-buffers/docs/proto3#reserved

@htuch Any other gotchas around removing fields from proto?